scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/hvson8.yaml

```diff
@@ -0,0 +1,44 @@
+HVSON-8-1EP_3.25x2.35mm_P0.8mm_E1.1x1.75mm:
+  device_type: 'HVSON'
+  library: Package_SON
+  #manufacturer: 'NXP'
+  size_source: 'https://www.nxp.com/docs/en/data-sheet/PCF8523.pdf (page 66)'
```

Comment:
Page 57

Reply:
ok   :(

Reply:
OK  IPC 7351 right so there is a reason for this. I'll search for that document, thanks